challenge.md

```diff
@@ -32,19 +32,20 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ It's an absolute path because it specifies several sub directories
```

Comment:
Right answer, wrong reason. It's an absolute path because it always refers to the same location, no matter where _you_ might happen to be.